tensorflow/python/ops/image_ops_impl.py

```diff
@@ -466,10 +466,10 @@ def flip_left_right(image):
     ValueError: if the shape of `image` not supported.
     
   Usage Example:
-    ```python
-    >> import tensorflow as tf
-    >> image = tf.random.normal(shape=(256, 256, 3))
-    >> tf.image.flip_left_right(image)
+    ```
+    >>> import tensorflow
```

Comment:
You don't need to import tensorflow. Just call `tf.<any function>`. The imports are 'inbuilt` into the test file.